dashboard/test/ui/features/teacher_tools/teacher_homepage.feature

```diff
@@ -35,6 +35,12 @@ Feature: Using the teacher homepage sections feature
 
   @properties_encryption_key
   Scenario: Navigate to course and unit pages
+
+    # This test uses many old features that are not in the new teacher homepage.
+    # Specifically, checking the hrefs of each page, the `.uitest-script-next-banner`
+    # and general navigation between course, unit and lesson pages.
+    # In order to turn this DCDO key on, we would need to re-write the test.
```

Comment:
This makes sense - love the comment.  I imagine this will also get a similar treatment once the homepage live.